base_app/restaurant.py

```diff
@@ -29,7 +30,8 @@ def from_json(cls, json_string):
                         r['phone'],
                         r['display_phone'],
                         r['rating'],
-                        r['price']
+                        r['price'],
+                        r['is_closed']
```

Comment:
It makes sense to add such an attribute, but I'm curious as to why you chose to go with `is_closed` as opposed to `open_now`.

Reply:
is_closed is the attribute being passed from the Yelp API, I just kept it the same.

Reply:
Thanks for explaining.